support/factories/gql/fetchOCR2KeyBundles.ts

```diff
@@ -0,0 +1,28 @@
+// buildOCR2KeyBundle builds a ocr2 key bundle for the FetchOCR2KeyBundles query.
+export function buildOCR2KeyBundle(
+  overrides?: Partial<Ocr2KeyBundlesPayload_ResultsFields>,
+): Ocr2KeyBundlesPayload_ResultsFields {
+  return {
+    __typename: 'OCR2KeyBundle',
+    id: '68ae4225aa9fd932e62a2411e4c757a9fa72de45426ac455801a6f08cb4392b3',
+    chainType: 'evm',
```

Comment:
You should change this to `EVM` to match the correct types

Reply:
Updating the graphql enums to remove terra and add starknet, the resolver, and will update the fetcher as well.  Thanks!

Reply:
Fixed, and the second key in the bundle now matches the new graphQL side (in the process of being merged in core) of 'STARKNET'

Reply:
It seems like starknet is not going to be merged any time soon, skipping it for now